dspace/config/spring/api/discovery.xml

```diff
@@ -61,6 +61,11 @@
                 <entry key="publicationVolume" value-ref="publicationVolume"/>
                 <entry key="periodical" value-ref="periodical"/>
                 <entry key="project" value-ref="project"/>
+                <!-- OpenAIRE4 guidelines - search for a creator that can be a Person
+                     or an OrgUnit -->
+                <entry key="openAIRECreatorConfiguration" value-ref="personOrOrganization"/>
```

Comment:
We've previously moved away from names containing suffix `Configuration` (see the lines above)
I would recommend to rename `openAIRECreatorConfiguration` to `openAIRECreator` here and in the submission forms.

Similar for `openAIREFundingAgencyConfiguration`

Reply:
Ok.